ROMFS/px4fmu_common/init.d/airframes/13012_convergence

```diff
@@ -69,4 +69,9 @@ param set-default VT_TYPE 1
 
 set MIXER vtol_convergence
 
-set PWM_OUT 1234
+if ! ver hwcmp MATEK_H743
```

Comment:
I don't like board specifics slipping into airframes like this, but we can live with it short term as it can all go away with control allocation soon. https://github.com/PX4/PX4-Autopilot/pull/19596

Reply:
This is the only approach at the moment to deal with the io timer change between pwm 1,2 and 3,4 of these boards. Even for a simple multicopter small changes in the defaults would be necessary.